### PR TITLE
show min/max line number in minibuffer

### PR DESCRIPTION
**Pull request by LiuYinCarl**
VScode can show the min/max line number of the file when goto line, it's a useful feature for users who want to know the total line number when use goto-line command, So I suggest to add the feature into `goto-line-preview`.

image

test with buffer which has contents

image

test with empty buffer

image

### CODE REVIEW

File: goto-line-preview.el [hunk 1]
```diff
@@ -87,9 +87,11 @@
         jumped)
     (run-hooks 'goto-line-preview-before-hook)
     (unwind-protect
-        (setq jumped (read-number (if goto-line-preview--relative-p
-                                      "Goto line relative: "
-                                    "Goto line: ")))
+        (setq jumped (read-number
+		      (let ((lines (count-lines (point-min) (point-max))))
```
Comment (jcs090218) [inline goto-line-preview.el]: Why not use `line-number-at-pos` with `(point-max)`? Just curious.

Comment (LiuYinCarl) [reply goto-line-preview.el]: @jcs090218 Thanks for your revirw, actually I'm not very familiar with elisp, if `(line-number-at-pos (point-max))` is better, I can edit the PR.

Comment (jcs090218) [reply goto-line-preview.el]: How about?

```elisp
(let ((lines (line-number-at-pos (point-max))))
  (format (if goto-line-preview--relative-p
              "[%d] Goto line relative: (%d to %d) "
            "[%d] Goto line: (%d to %d) ")
          goto-line-preview--prev-line-num
          (max 0 (min 1 lines))
          lines))
```

I've changed the format a little and added the current line in the front. Example output:

```
[13] Goto line: (1 to 100) 
```

P.S. It seems like there are indention error there, can you fix that for us?

Comment (LiuYinCarl) [reply goto-line-preview.el]: @jcs090218 Thanks for your advise, I fix the code, would please review again?

**Review by jcs090218 — APPROVED** [review]
I've saw you request me to review, I assumed you are okay with my suggestions? :)

Thank your for your contribution! And thanks for the idea, it's very nice to have! ❤️ 🚀

**Comment by LiuYinCarl** [comment]
@jcs090218 Your suggestion is good and make the code more clean!

I have another suggestion which is highlight the selected line(line which need to preview) for a short time(such as 0.5 second), because it's hard to fast find the line wanted to jump if somebody close `line-mode` or is using a big screen. I tried but don't know how to do it.  

Just a suggestion.

**Comment by jcs090218** [comment]
I agree! I was expecting the `hl-line-mode` would achieve something close to this, but it seems like it doesn't update from the minibuffer for some reason. Or try [beacon](https://github.com/Malabarba/beacon) (I personally don't use it but group of people seems to like it)?

I would love to see this feature, as always, issues and PRs are welcome! 😄 👍

**Pull request merged**